sockeye/inference.py

```diff
@@ -1309,37 +1428,48 @@ def _make_result(self,
                      trans_input: TranslatorInput,
                      translation: Translation) -> TranslatorOutput:
         """
-        Returns a translator result from generated target-side word ids, attention matrix, and score.
+        Returns a translator result from generated target-side word ids, attention matrices and scores.
         Strips stop ids from translation string.
 
         :param trans_input: Translator input.
         :param translation: The translation + attention and score.
         :return: TranslatorOutput.
         """
         target_ids = translation.target_ids
+        target_tokens = utils.ids_to_tokens(translation.target_ids, self.vocab_target_inv)
+        target_string = utils.tokens_to_string(target_ids, target_tokens, self.strip_ids)
```

Comment:
this PR is probably going on top of the scoring PR from @mjpost . He also added a function like this. Please make sure that this is properly merged/unified.

Reply:
Sure. Then I should probably wait until the scoring PR is merged.